src/nspawn/nspawn-mount.c

```diff
@@ -271,7 +275,13 @@ int mount_sysfs(const char *dest) {
                 return log_error_errno(errno, "Failed to remove %s: %m", full);
 
         x = prefix_roota(top, "/fs/kdbus");
-        (void) mkdir(x, 0755);
+        (void) mkdir_p(x, 0755);
```

Comment:
`/sys/fs` doesn't exist at this point so the previous simple `mkdir()` call would fail.